buildSrc/src/main/kotlin/com/getstream/sdk/chat/Dependencies.kt

```diff
@@ -97,6 +98,7 @@ object Dependencies {
     const val androidxViewPager2 = "androidx.viewpager2:viewpager2:$ANDROIDX_VIEW_PAGER_2"
     const val assertj = "org.assertj:assertj-core:$ASSERTJ_VERSION"
     const val awaitility = "org.awaitility:awaitility:$AWAITILITY_VERSION"
+    const val burst = "com.squareup.burst:burst-junit4:$BURST_VERSION"
```

Comment:
[Burst](https://github.com/square/burst) is a type safe helper for parameterized tests, by Square